packages/swapper/src/manager/SwapperManager.test.ts

```diff
@@ -227,11 +228,13 @@ describe('SwapperManager', () => {
 
       const cowSwapperGetUsdTradeQuoteMock = jest
         .spyOn(cowSwapper, 'getTradeQuote')
-        .mockImplementation(jest.fn().mockResolvedValueOnce(badTradeQuote))
+        // TODO(gomes): is mocking OK(badTradeQuote) what we actually want here, in other words what's the intent of this?
```

Comment:
excellent comment

Reply:
Ty for the context ser, that's precisely what I needed to know 🙏🏽 972240287e57a37158b162d8a1123f638f00b278